etc/vlc.profile

```diff
@@ -23,7 +23,7 @@ protocol unix,inet,inet6,netlink
 seccomp
 shell none
 
-private-bin vlc,cvlc,nvlc,rvlc,qvlc,svlc,vlc-wrapper
+private-bin vlc,cvlc,nvlc,rvlc,qvlc,svlc
```

Comment:
That's my mistake. I wanted to add it first but reconsidered after reading [manual](https://manpages.debian.org/stretch/vlc-bin/vlc-wrapper.1.en.html). It needs sudo so it won't work with firejail anyway and it's very special case usage which 99% users won't care about.

Unfortunately I committed changes with `vlc-wraper` added to private-bin mistakenly so I had  to remove it in next commit. I'm sorry for this ugliness.